Dockerfile

```diff
@@ -11,14 +11,22 @@ RUN apt-get -qq update && \
     php7.0-mcrypt php7.0-json php7.0-curl php7.0-zip php7.0-gd \
     pdftk a2ps htmldoc mysql-client \
     php7.0-mbstring \
-    php-xdebug ssh php-common \
+    php-xdebug ssh php-common curl \
     && rm -rf /var/lib/apt/lists/*
 
 # replace the php memcache with a fixed one
 COPY php-memcache/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb /home/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb
 RUN dpkg -i /home/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb \
     && echo php-memcache hold | dpkg --set-selections \
     && rm /home/php-memcache_3.0.9-20151130.fdbd46b-2.flash1_amd64.deb
+
+# install node
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - && \
+    apt-get update && \
```

Comment:
Can't you avoid to do another `apt-get update` 2min after the first one above?
Maybe you should add the repository before the first `apt-get update` so you can only do one update